entities/params/postforgotpassword.go

```diff
@@ -0,0 +1,10 @@
+package params
```

Comment:
@Dzalevski these params should be put in the `users.go` file, we should not do separate file for each param struct. We'll have too many files.